deepmd/descriptor/se_a.py

```diff
@@ -226,6 +230,37 @@ def compute_input_stats (self,
             self.davg = np.array(all_davg)
         self.dstd = np.array(all_dstd)
 
+    def enable_compression(self,
+                           distance,
+                           max_nbor_size,
+                           env_mat_range,
+                           model_file = 'frozon_model.pb',
+                           table_config = [5, 0.01, 0.1, -1]
+    ) -> None:
+        """
+        Reveive the statisitcs (distance, max_nbor_size and env_mat_range) of the training data.
+        
+        Parameters
+        ----------
+        distance
```

Comment:
change `distance`  to `min_nbor_dist` ?  Isn't it a variable that is never referred?